maps/CEVEris/_CEV_Eris.dmm

```diff
@@ -74165,6 +74186,11 @@
 /obj/machinery/atmospherics/pipe/simple/hidden/supply{
 	dir = 4
 	},
+/obj/item/weapon/gun/projectile/handmade_pistol{
+	step_x = 0;
```

Comment:
This breaks everything, remove the edited step vars.

Reply:
I suggest just editing the file directly, and deleting the two lines instead of using the map editor.

Reply:
oh thanks i'll do it right now